### PR TITLE
AISDK-116: Add GET list of custom vocabularies to examples

### PR DESCRIPTION
**Pull request by k-weng**

### CODE REVIEW

File: examples/submit_custom_vocabularies.js [hunk 1]
```diff
@@ -46,6 +46,12 @@ const token = require('./config/config.json').access_token;
             , details: ${cv_submission.failure_detail}`)
     }
 
+    /**
+     * Gets list of custom vocabularies
+     */
+    console.log("Getting list of submitted custom vocabularies.");
+    var listCustomVocabularieInformations = await client.getListOfCustomVocabularyInformations();
```
Comment (KostasRev) [inline examples/submit_custom_vocabularies.js]: Minor - Typo in var name

File: examples/submit_custom_vocabularies.js [hunk 2]
```diff
@@ -46,6 +46,12 @@ const token = require('./config/config.json').access_token;
             , details: ${cv_submission.failure_detail}`)
     }
 
+    /**
+     * Gets list of custom vocabularies
```
Comment (KostasRev) [inline examples/submit_custom_vocabularies.js]:
```suggestion
     * Gets list of custom vocabularies' information
```

Trying to clarify that this gets the information here, just like we are clear about it in the client method name.

**Review by KostasRev — CHANGES_REQUESTED** [review]
A couple of minor comments, but otherwise looks good.

**Review by amikofalvy — APPROVED** [review]

**Review by KostasRev — APPROVED** [review]
Looks good to me!

**Pull request merged**